Index out-of-bounds error during html generation

I've been running thyme all day and tried to generate the html report, but got an error:

```
[17:24:26] ~ $ thyme show -i thyme.json -w stats > thyme.html
panic: runtime error: index out of range

goroutine 1 [running]:
panic(0x1f4ee0, 0xc820012090)
	/usr/local/Cellar/go/1.6.3/libexec/src/runtime/panic.go:481 +0x3e6
github.com/sourcegraph/thyme.(*Window).Info(0xc820365920, 0xc8201e9920)
	/Users/arthur/Projects/go/src/github.com/sourcegraph/thyme/data.go:130 +0x2bc
github.com/sourcegraph/thyme.appID(0xc820365920, 0x0, 0x0)
	/Users/arthur/Projects/go/src/github.com/sourcegraph/thyme/show.go:381 +0x42
github.com/sourcegraph/thyme.NewTimeline(0xc8200b6740, 0x297bc8, 0xc8203e8e40)
	/Users/arthur/Projects/go/src/github.com/sourcegraph/thyme/show.go:177 +0x983
github.com/sourcegraph/thyme.Stats(0xc8200b6740, 0x0, 0x0)
	/Users/arthur/Projects/go/src/github.com/sourcegraph/thyme/show.go:18 +0x6d
main.(*ShowCmd).Execute(0x355e20, 0xc8200aea50, 0x0, 0x5, 0x0, 0x0)
	/Users/arthur/Projects/go/src/github.com/sourcegraph/thyme/cmd/thyme/main.go:114 +0x59e
github.com/jessevdk/go-flags.(*Parser).ParseArgs(0xc8200c8000, 0xc82000a0d0, 0x5, 0x5, 0x0, 0x0, 0x0, 0x0, 0x0)
	/Users/arthur/Projects/go/src/github.com/jessevdk/go-flags/parser.go:314 +0xa83
github.com/jessevdk/go-flags.(*Parser).Parse(0xc8200c8000, 0x0, 0x0, 0x0, 0x0, 0x0)
	/Users/arthur/Projects/go/src/github.com/jessevdk/go-flags/parser.go:185 +0x9b
main.main.func1(0x0, 0x0)
	/Users/arthur/Projects/go/src/github.com/sourcegraph/thyme/cmd/thyme/main.go:141 +0x2f
main.main()
	/Users/arthur/Projects/go/src/github.com/sourcegraph/thyme/cmd/thyme/main.go:148 +0x23
```

The patch referenced above appears to have worked for issues related to Chrome, but I am still seeing the same issue after running multiple other apps and attempting to generate html with thyme show -i thyme.json -w stats > thyme.html

The error I get is: 
panic: runtime error: index out of range

--- a/data.go
+++ b/data.go
@@ -125,10 +125,18 @@ func (w *Window) Info() *Winfo {
 	first := strings.TrimSpace(fields[0])
 	last := strings.TrimSpace(fields[len(fields)-1])
 	if last == "Google Chrome" {
-		return &Winfo{
-			App:    "Google Chrome",
-			SubApp: strings.TrimSpace(fields[len(fields)-2]),
-			Title:  strings.Join(fields[0:len(fields)-2], " - "),
+		if len(fields) > 1 {
+			return &Winfo{
+				App:    "Google Chrome",
+				SubApp: strings.TrimSpace(fields[len(fields)-2]),
+				Title:  strings.Join(fields[0:len(fields)-2], " - "),
+			}
+		} else {
+			return &Winfo{
+				App:    "Google Chrome",
+				SubApp: "",
+				Title:  "",
+			}
 		}
 	} else if first == "Slack" {
 		return &Winfo{

